test/FeatureFlagTest.php

```diff
@@ -20,10 +20,10 @@ class FeatureFlagTest extends TestCase
     private $http_client;
     private $client;
 
-    public function setUp($decideEndpointResponse = MockedResponses::DECIDE_V3_RESPONSE, $personalApiKey = "test"): void
+    public function setUp($flagsEndpointResponse = MockedResponses::DECIDE_V3_RESPONSE, $personalApiKey = "test"): void
```

Comment:
Should we rename the `DECIDE_V3_RESPONSE` type to `FLAGS_V2_RESPONSE`?

Reply:
yup good call